### ISSUE TITLE
Honeybadger error - Errno::ENOENT: No such file or directory @ rb_sysopen - /opt/lib-jobs/shared/staff-directory_20220624.csv

### ISSUE THREAD

**Issue by christinach** [pulibrary/lib_jobs#376]
[6/25/2022 at 7:30 AM -04:00](https://app.honeybadger.io/projects/100710/faults/86337928/01G6DA0H0ZZWGPD01Y74T7P7NP?page=0)

https://app.honeybadger.io/projects/100710/faults/86337928

Errno::ENOENT: No such file or directory @ rb_sysopen - /opt/lib-jobs/shared/staff-directory_20220624.csv

app/controllers/staff_directory_controller.rb
```
def removed
  today_report = File.new(WebStaff::StaffDirectoryGenerator.report_filename)
  yesterday_report = File.new(WebStaff::StaffDirectoryGenerator.yesterday_filename)
  differ = WebStaff::StaffDirectoryDifference.new(new_report: today_report, old_report: 
  yesterday_report)
  respond_to do \|format\|
```

Note: This should only have affected any systems that utilize the `removed-staff.txt` report, and @carolyncole has shared that this is a deprecated report that is not being used.

**Comment by christinach** [comment]
because of the missing file `Errno::ENOENT in rake lib_jobs:generate_staff_report --silent` was triggered with Honeybadger error: https://app.honeybadger.io/projects/100710/faults/86337710

**Comment by carolyncole** [comment]
@christinach @maxkadel I believe that nothing utilizes the removed file at this point.  It would be nice if the rake task did not fail, but instead logged an error.  That said it should have no impact.  We intended for removed staff to be utilized to disable users from the web page, but we realized that folks get added and removed from the PeopleSoft report for reasons other than termination.  For example when a person changes organization structure they can be removed from the file for a few days.  This happened to Wind C. during our testing phase.
